pom.xml

```diff
@@ -84,8 +84,8 @@ THE SOFTWARE.
     <project.patchManagement.url>https://api.github.com</project.patchManagement.url>
     <patch.tracker.serverId>jenkins-jira</patch.tracker.serverId>
 
-    <animal.sniffer.skip>${skipTests}</animal.sniffer.skip>
-    <java.level>8</java.level>
+    <animal.sniffer.skip>true</animal.sniffer.skip>
```

Comment:
Why?

As of Java 9 (IIRC) and availability of `-release` we could consider dropping Animal Sniffer entirely. Its main benefit is being able to use an annotation to allow a particular method to use APIs from a newer Java Platform version but that is pretty fragile. If such a trick is really needed in a limited case, we can use reflection. (Prefer to avoid multiversion JARs. Overengineering.)

Reply:
> Why?

Have you tried deleting `<animal.sniffer.skip>true</animal.sniffer.skip>` and compiling the code to see what happens?

Reply:
I think what Jesse means is why is animal sniffer still here? I assume your comment @basil is because it's automatically added as part of the parent pom and needs removing from there?

https://github.com/jenkinsci/pom/blob/dc95d243aafe8778bb86ef3f446dad493091d58c/pom.xml#L731-L749

i.e. it should either only be added on Java 8 as part of a profile or just removed from the parent pom

Reply:
> just removed from the parent pom

You mean like I already did in jenkinsci/pom#209? I fail to see how [reviews](https://github.com/jenkinsci/jenkins/pull/6083#pullrequestreview-838548606) whose only action items are ones I have already performed are adding value to the process.

Reply:
A TODO comment linking to the PR would’ve mean the review wasn’t needed

Reply:
> A TODO comment linking to the PR

Done